### PR TITLE
Add danish locale to reports

### PR DESCRIPTION
**Pull request by Hyllegaard**

### CODE REVIEW

**Comment by sni** [comment]
nice, while you're finalizing the pull request, could you rename the language file to .dk?

**Comment by Hyllegaard** [comment]
Things should be ok now. I was not aware of all the files that needed to be changed :)

**Comment by sni** [comment]
no problem, thats why we have tests :-)

**Comment by Hyllegaard** [comment]
Is there something I have to do, to finalize it?

File: plugins/plugins-available/reports2/templates/reports/locale/dk.tt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    'Unreachable'                           => 'Unreachable',
+    'Ok'                                    => 'Ok',
+    'Warning'                               => 'Warning',
+    'Unknown'                               => 'Unknown',
+    'Critical'                              => 'Critical',
+    'Host:'                                 => 'Host:',
+    'Service:'                              => 'Service:',
+
+# translated patterns
+    'Hostgroup:'                            => 'Hostgruppe:',
```
Comment (sni) [inline plugins/plugins-available/reports2/templates/reports/locale/dk.tt]: are these 2 lines really danish?

**Comment by Hyllegaard** [comment]
Yes, gruppe is spelled the same way in german and danish.

**Comment by sni** [comment]
funny :-) then thanks for this translation

**Comment by Hyllegaard** [comment]
You are more than welcome. It's just a tiny contribution to a great project.

**Pull request merged**